nixos/modules/services/misc/gitea.nix

```diff
@@ -367,15 +375,17 @@ in
       };
     };
 
-    users = mkIf (cfg.user == "gitea") {
-      users.gitea = {
+    users.users = mkIf (cfg.user == "gitea") {
+      gitea = {
         description = "Gitea Service";
         home = cfg.stateDir;
-        createHome = true;
         useDefaultShell = true;
+        group = "gitea";
       };
     };
 
+    users.groups.gitea = {};
```

Comment:
previously, gitea was running as group "nogroup", wasn't it?

We should probably keep that default, also w.r.t. state files laying around

Reply:
@flokli you are correct in that `gitea` is currently running as `nogroup`. Please see https://github.com/NixOS/nixpkgs/issues/55370. Maybe I should add a new entry to `systemd.tmpfiles.rules` such that all data is recursively `chgrp`'d to `gitea`...

Reply:
makes sense, let's do that!

Reply:
Thanks for the feedback! Applied.